Cargo.toml

```diff
@@ -57,7 +57,7 @@ members = [
 ]
 
 [workspace.metadata.workspaces]
-version = "0.14.0"
+version = "0.15.0"
```

Comment:
Lgtm. Although, I wonder if the new workspace inheritance feature that `rustc 1.64.0`'s `cargo` ships with could help us here.

https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-package-table

But at least for now, this looks good!